index.js

```diff
@@ -123,17 +124,24 @@ module.exports = function (host_or_multiaddr, port) {
     var file
 
     for (var i = 0; i < files.length; i++) {
+
       file = files[i]
-      if (file instanceof stream.Stream || Buffer.isBuffer(file)) {
+
+      if (Buffer.isBuffer(file)) {
+        file = new File({
+          cwd: '/',
+          base: '/',
+          path: '/',
+          contents: streamifier.createReadStream(file)
+        })
+      } else if (file instanceof stream.Stream) {
         file = new File({
           cwd: '/',
           base: '/',
           path: '/',
           contents: file
         })
-      }
```

Comment:
goes on like:
```
else if (!file instanceof File) {
        return null
      }

      files[i] = file
    }
```

Just passthrough, strings, like local paths, should just passthrough.